devsetup/scripts/tripleo.sh

```diff
@@ -101,14 +96,13 @@ export OCTAVIA_ENABLED=${OCTAVIA_ENABLED}
 export TELEMETRY_ENABLED=${TELEMETRY_ENABLED:-true}
 
 set +x
-if [[ -f \$HOME/containers-prepare-parameters.yaml ]]; then
-    echo "Using existing containers-prepare-parameters.yaml - contents:"
-    cat \$HOME/containers-prepare-parameters.yaml
-else
+if [ ! -f \$HOME/containers-prepare-parameters.yaml ]; then
     login_args=" "
     [ "\$RH_REGISTRY_USER" ] && [ -n "\$RH_REGISTRY_PWD" ] && login_args="--enable-registry-login"
+    set -x
```

Comment:
will this expose the login args that are placed in the containers-prep file?
i understand you had to add these for local runs but one of the main goals of this work is to add a multicell job. In that case (the ci job) will you just use the existing registry login (that comes from the config repo playbooks and used in the other adoption jobs)?

Reply:
no, as ``login_args="--enable-registry-login"`` and contains no creds.

Reply:
In CI, there would be no RH_REGISTRY creds, so it would work by usual for CI meanings.

Reply:
I got now that you mean, removed ``-x``